nextflow.config

```
@@ -17,6 +17,23 @@ params {
     igenomes_base              = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore            = false
 
+    // Taxonomy classification options
+    classify_taxonomy                       = true
+    taxonomy_mmseqs_save_intermedfiles      = false
+
+    taxonomy_mmseqs_databases_localpath     = null
+    taxonomy_mmseqs_databases_id            = 'SILVA'
+    taxonomy_mmseqs_databases_savetmp       = false
+
+    taxonomy_mmseqs_taxonomy_savetmp        = false
+    taxonomy_mmseqs_taxonomy_searchtype     = 2
+    taxonomy_mmseqs_taxonomy_lcaranks       = 'kingdom,phylum,class,order,family,genus,species'
+    taxonomy_mmseqs_taxonomy_taxlineage     = 1
+    taxonomy_mmseqs_taxonomy_sensitivity    = 5.0
+    taxonomy_mmseqs_taxonomy_orffilters     = 2.0
```

```suggestion
    taxonomy_mmseqs_taxonomy_sensitivity    = 5
    taxonomy_mmseqs_taxonomy_orffilters     = 2
```

Can we have integers here, not float? Otherwise linting is complaining because it compares these values against the `schema.json` where you put these values as integers.

---

im afriad the values can be 7.5 and 5.5 so floats and when i add them as floats it complains and changes it to integer. Im linting it now and will see hot that goes

---

Yep, I saw you changed to string instead, that should do the trick :)